### PR TITLE
Change memory map size & fix up styling

### PR DESCRIPTION
**Pull request by Geod24**
Just a few refactoring we can already integrate.

### CODE REVIEW

File: source/agora/common/BlockStorage.d [hunk 1]
```diff
@@ -34,16 +34,16 @@ import std.stdio;
```
Comment (AndrejMitrovic) [inline source/agora/common/BlockStorage.d:34]: Typo in commit message: `sytle`

File: source/agora/common/BlockStorage.d [hunk 2]
```diff
@@ -43,7 +43,7 @@ private immutable size_t ReserveSize = 64 * 1024;
 private immutable ulong MaxBlock = 100;
 
 /// The map file size
-private immutable size_t MapSize = 640 * 1024;
+private immutable size_t MapSize = 64 * 1024;
```
Comment (AndrejMitrovic) [inline source/agora/common/BlockStorage.d:46]: The commit message should explain the reason why this change was made.

**Comment by AndrejMitrovic** [comment]
Just two issues, otherwise LGTM.

File: source/agora/common/BlockStorage.d (hunk 2 above)
Comment (Geod24) [reply source/agora/common/BlockStorage.d:46]: @MukeunKim : What would be a good commit message ?

Comment (Geod24) [reply source/agora/common/BlockStorage.d:46]: Fixed

**Review by AndrejMitrovic — APPROVED** [review]
